picard/ui/infodialog.py

```diff
@@ -150,11 +163,28 @@ class ArtworkTableExisting(ArtworkTable):
     artwork_columns = ('orig', 'new',)
 
 
+class ArtworkTableExistingExternal(ArtworkTable):
+    NUM_COLS = 4
+
+    _columns = {
+        'orig': 0,
+        'type': 1,
+        'new': 2,
+        'external': 3,
+    }
+
+    _labels = (_("Existing Cover"), _("Type"), _("New Cover"), _("New External Cover"),)
```

Comment:
We could improve column titles and/or add a tool tip to clarify what are those exactly.

Reply:
Yes. I think we need better naming, but not sure about the exact wording. As we have "External Cover" the other should probably be "Tag Cover". It's all not really self-explaining, though. But yes, a tooltip could help with that.